src/internal/m365/collection/drive/collections_tree_test.go

```diff
@@ -588,11 +338,14 @@ func (suite *CollectionsTreeUnitSuite) TestCollections_PopulateTree_singleDelta(
 		{
 			name: "nil page",
 			tree: newFolderyMcFolderFace(nil, rootID),
-			enumerator: mock.EnumerateItemsDeltaByDrive{
-				DrivePagers: map[string]*mock.DriveItemsDeltaPager{
+			enumerator: mock.EnumerateDriveItemsDelta{
+				DrivePagers: map[string]*mock.DriveDeltaEnumerator{
 					id(drive): {
-						Pages:       nil,
-						DeltaUpdate: pagers.DeltaUpdate{URL: id(delta)},
+						DriveID: id(drive),
+						DeltaQueries: []*mock.DeltaQuery{{
+							Pages:       nil,
+							DeltaUpdate: pagers.DeltaUpdate{URL: id(delta)},
```

Comment:
nit: can this be swapped with `mock.Delta(id(delta), nil)`?

Reply:
No can do.  It's a special case that needs an explicit struct creation for the test.  I'll make a note of it.  As for the mock.Delta, it's not the same item type.